packages/firebase_auth/firebase_auth/android/src/main/java/io/flutter/plugins/firebaseauth/FirebaseAuthPlugin.java

```diff
@@ -62,7 +63,7 @@
 public class FirebaseAuthPlugin implements MethodCallHandler, FlutterPlugin, ActivityAware {
   // Only set registrar for v1 embedder.
   private PluginRegistry.Registrar registrar;
-  private SparseArray<AuthStateListener> authStateListeners;
+  SparseArray<PluginAuthStateListener> authStateListeners;
```

Comment:
Had to expose this for tests

Reply:
What do you think about keeping this private and adding a @VisibleForTestings method: `GetAuthStateListeners` to use for testing?

Reply:
I'll try this :)